WORKSPACE

```diff
@@ -168,10 +168,6 @@ http_archive(
 http_archive(
     name = "jsonhpp",
     build_file = "//bazel:jsonhpp.BUILD",
```

Comment:
I think you can remove both this line:

```suggestion
```

and delete the file `bazel/jsonhpp.BUILD` since the archive already contains the BUILD file with the same target.

Reply:
My bad :) Nice!